src/mlnet/mlnet.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Can we upgrade this to net core 3.0? it should be simply a one - line change @maryamariyan @JakeRadMSFT 

I know that the current recommend way from dotnet is to downgrade to netcore 2.1. But we continuously received issues from customers complaining about cli can't run on netcore 3.0. Even after hint message about downgrading was shown.

Reply:
Let's do that in a separate change from porting the code. Can you please log an issue for it?

The drawback is that once you upgrade to `3.0`, you are requiring users to install `3.0`. But `2.1` is the current long-term support version.

Reply:
@eerhardt Can we support both somehow?

Reply:
No matter what, I feel this is a separate discussion from merging the branch into `master`.

I'm not sure how to support both. It may be possible, but that may involve duplicating all the assemblies and doubling the size of the NuGet package.

Reply:
As Eric says, this can be a separate discussion.

The example on https://docs.microsoft.com/en-us/dotnet/standard/frameworks may work.

> The following library project file targets APIs of .NET Standard (netstandard1.4) and APIs of the .NET Framework (net40 and net45). Use the plural **TargetFrameworks** element with multiple target frameworks. Note how the Condition attributes include implementation-specific packages when the library is compiled for the two .NET Framework TFMs:
> 
> ```xml
> <Project Sdk="Microsoft.NET.Sdk">
> 
>   <PropertyGroup>
>     <TargetFrameworks>netstandard1.4;net40;net45</TargetFrameworks>
>   </PropertyGroup>
> 
>   <!-- Conditionally obtain references for the .NET Framework 4.0 target -->
>   <ItemGroup Condition=" '$(TargetFramework)' == 'net40' ">
>     <Reference Include="System.Net" />
>   </ItemGroup>
> 
>   <!-- Conditionally obtain references for the .NET Framework 4.5 target -->
>   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
>     <Reference Include="System.Net.Http" />
>     <Reference Include="System.Threading.Tasks" />
>   </ItemGroup>
> 
> </Project>
> ```
> Within your library or app, you write conditional code to compile for each target framework:
> ```c#
> public class MyClass
> {
>     static void Main()
>     {
> #if NET40
>         Console.WriteLine("Target framework: .NET Framework 4.0");
> #elif NET45  
>         Console.WriteLine("Target framework: .NET Framework 4.5");
> #else
>         Console.WriteLine("Target framework: .NET Standard 1.4");
> #endif
>     }
> }
> ```

I think ours would be:
```suggestion
    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```